docs/pages/beacon-management/networking.md

```diff
@@ -68,22 +72,20 @@ Libp2p operates at the lower levels of the OSI model, particularly at the Transp
 
 If your setup is behind a firewall there are a few ports that will need to be opened to allow for P2P discovery and communication. There are also some ports that need to be protected to prevent unwanted access or DDOS attacks on your node.
 
-Ports that should be opened:
+Ports that must be opened:
 
-- 30303/TCP+UDP - Execution layer p2p communication port
-- 9000/TCP+UDP - Beacon Node P2P communication port
-- 9090/TCP - Lodestar IPv6 P2P communication port
-- 13000/TCP - Prysm P2P communication port
-- 12000/UDP - Prysm P2P communication port
```

Comment:
These should not be removed or lodestar will have issue p2p communicating with Prysm.  I get that these are Lodestar docs but we communicate with Prysm

https://docs.prylabs.network/docs/prysm-usage/p2p-host-ip#configure-your-firewall

Reply:
Perhaps they go in an "optional" section with a note so people know the issue exists.

Reply:
> lodestar will have issue p2p communicating with Prysm

This only concerns _inbound_ ports, not _outbound_. I already mentioned that in the PR description, if you start speculating on what ports other nodes in the network are running on, you are fighting a losing battle.

Just an example, on dappnode [Lodestar p2p port is configured to use 9512](https://github.com/dappnode/DAppNodePackage-Lodestar-Prater/blob/3490e356b2a041355a8e7d50ae7e01bf436108f4/docker-compose.yml#L21), same goes for other clients, they don't use default ports.

Maybe we should highlight that it is important to not block **any** outbound ports, but I felt like this is unnecessary as nobody does this.

Reply:
OK makes sense.  Maybe we can add a note in another PR that these are the defaults and that there are some extenuating circumstances that do not fall under these rules

Reply:
Noticed in the prysm docs you linked above that they state to allow outbound traffic from any port
![image](https://github.com/ChainSafe/lodestar/assets/38436224/08102a6d-f232-41bb-8646-8ef14d028d18)
This might be something to consider but from my experience this is not an issue as there is no guide that I know that instructions users to block outbound traffic.